src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -64,14 +65,26 @@ const WorkspacePermissionNotice = ({ workspace }) => {
   ])
 }
 
+const AzureWarning = () => {
+  const warningMessage = 'Reminder: Azure workspaces (like this one) are currently NOT authorized to contain any federally protected data. ' +
+    'Please do not import or use any federally protected data in Azure workspaces.'
+
+  return h(TitleBar, {
+    title: div({ style: { display: 'flex', alignItems: 'center', margin: '1rem' } }, [
+      icon('warning-standard', { size: 32, style: { color: colors.danger(), marginRight: '0.5rem' } }),
```

Comment:
I personally think a smaller font size looks better.

Reply:
font size was defaulting to 18. here's 16 and 14:
![Screen Shot 2022-12-02 at 10 14 30 AM](https://user-images.githubusercontent.com/5375000/205324909-55683a1c-1304-451b-b5f4-9ec6e48dc8bd.png)
![Screen Shot 2022-12-02 at 10 14 15 AM](https://user-images.githubusercontent.com/5375000/205324911-b2a1cbb4-afac-43fb-b7c1-d86a1735486a.png)

I think 14 does look nicer. I'm interested to know if you think 14 or 16 is better!

Reply:
14 seems reasonable.